tools/get_dicom.pl

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/perl -w 
+use strict;
+
+=pod
+
+=head1 NAME
+
+get_dicom.pl - extracts DICOM files for specific patient names/scan types
```

Comment:
I would probably tend to name the script `get_dicom_files.pl` to be a little bit clearer on what this does.